pages/ui-components/packages.md

```diff
@@ -8,113 +8,164 @@ type: docs
 subnav:
```

Comment:
Lead should pluralize "component" 
`USWDS component packages allow you to import only the components your project needs.`